README.md

```diff
@@ -161,7 +161,7 @@ Our [stability guide] helps to judge how much churn can be expected when dependi
 
 ### Download a Binary Release
 
-Using `cargo binstall`, one is able to fetch [binary releases][releases]. You can install it via `cargo install cargo-quickinstall`, assuming 
+Using `cargo binstall`, one is able to fetch [binary releases][releases]. You can install it via `cargo install cargo-binstall`, assuming 
```

Comment:
You can link to the [`cargo-binstall` installation sector](https://github.com/cargo-bins/cargo-binstall/#installation) which provides binary installation.

Sorry I forgot this when editing the file.

Reply:
And I missed it in the review :/!